packages/grafana-data/src/themes/createPalette.ts

```diff
@@ -51,8 +51,8 @@ export interface ThemePaletteBase<TColor> {
 export interface ThemePalette extends ThemePaletteBase<ThemePaletteColor> {
   /** Returns a text color for the background */
   getContrastText(background: string): string;
-  /* Retruns a hover color for any default color */
-  getHoverColor(defaultColor: string): string;
+  /* Return a hover color for any default color */
+  getHoverColor(defaultColor: string, hoverFactor?: number): string;
```

Comment:
Nit: 
```suggestion
  /* Return a hover color for any color */
  getHoverColor(color: string, hoverFactor?: number): string;
```

Not sure what "default" stands for here